impl/src/main/java/io/kroxylicious/testing/kafka/testcontainers/TestcontainersKafkaCluster.java

```diff
@@ -276,13 +280,45 @@ private synchronized String buildBrokerList(Function<Integer, KafkaClusterConfig
                 .collect(Collectors.joining(","));
     }
 
+    /**
+     * Gets the container image version tag for the specified kafka version.
+     * This knows about the naming conventions used by the ogunalp native image project.
+     *
+     * @param kafkaVersion desired kafka version, which may be null.
+     * @return image tag or null
+     */
+    @Nullable
```

Comment:
I'm a bit confused as to what a null return value is supposed to mean. Particularly in the latest release case.

Reply:
Yeah I don't think we need to offer an i dependant override for zookeeper, though I do fully understand your line of thinking.

Reply:
I was thinking about the use-case where a developer wants to override say the kafka native with her own, but leave the zookeeper one using the default.   But like I say, I really don't think this is for this PR or now.